lib/user.js

```diff
@@ -52,6 +76,46 @@ class UserModel extends BaseModel {
     unsealToken(sealed, callback) {
         return iron.unseal(sealed, this.password, iron.defaults, callback);
     }
+
+    /**
+     * Get permissions on a specific repo
+     * @param  {String}  config.username        Username
+     * @param  {String}  config.scmUrl          The scmUrl of the repository
+     * @return {Function} callback              fn(err, permissions) where permissions is an object
+     *                                          that contains the permissions for [admin, push, pull]
+     *                                          Example: {admin: false, push: true, pull: true}
+     */
+    getPermissions(config, callback) {
+        const id = this.generateId({ username: config.username });
+
+        async.waterfall([
+            (next) => {
+                this.get(id, next);
+            },
+            (user, next) => {
+                this.unsealToken(user.token, next);
+            },
+            (unsealed, next) => {
+                const matched = (schema.config.regex.SCM_URL).exec(config.scmUrl);
+                const breaker = new Breaker(command);
```

Comment:
Because this is defined inside the scope of getPermissions, I don't think this will work globally.  @nkatzman can you comment here on the best way to use it